### PR TITLE
feat(): Add Splitbee analytics for pass downloads

### PR DESCRIPTION
**Pull request by luisivan**
PS: On a different note, we should add Prettifier config files to all projects to avoid noise in PRs

### CODE REVIEW

**Review by aahna-ashina — CHANGES_REQUESTED** [review]
@luisivan This PR is both adding Splitbee _and_ doing a lot of refactoring at the same time, so it's not easy for code reviewers to read the Splitbee code diff.  Can you either 1) create one PR per change (one PR for refactoring, and another for Splitbee), or 2) create one PR with one commit per change (one commit for refactoring, and another for Splitbee)?  Thanks! 😃

(Here is a great pull request philosophy: https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#pull-request-philosophy)

**Comment by luisivan** [comment]
Yeah... that's what I was mentioning above, since we didn't have a Prettifier config, just doing a save on my editor refactored the whole thing 😄 

Maybe it's best to first merge https://github.com/nation3/mobile-passport/pull/34, and then this one after (and then the changes will only affect Splitbee)

**Comment by luisivan** [comment]
@aahna-ashina ready to review again!

File: server/pages/api/downloadPass.ts [hunk 1]
```diff
@@ -7,6 +7,10 @@ import PassportIssuer from '../../abis/PassportIssuer.json'
 import Passport from '../../abis/Passport.json'
 import { ethers } from 'ethers'
 
+const { SplitbeeAnalytics } = require('@splitbee/node')
+
+const analytics = new SplitbeeAnalytics(process.env.SPLITBEE_TOKEN)
```
Comment (aahna-ashina) [inline server/pages/api/downloadPass.ts:12]: If the `SPLITBEE_TOKEN` is missing from `.env.local`, will the code still run on localhost?

**Review by aahna-ashina — APPROVED** [review]
Looks good to me (though I have not tested the changes on `localhost` yet.

Out of curiosity:  Where will the collected analytics be made available? On the Splitbee website I'm assuming?

**Comment by aahna-ashina** [comment]
@luisivan This PR will be closed when we merge #73 because we can get the same pass details by querying the `downloads` table that was added:  https://github.com/nation3/mobile-passport/pull/73/commits/90afce23bd6dfbc4330c404505d8671b4719cc7e

```sql
select time_of_download, platform, address, passport_id from downloads;
```

**Pull request closed without merging**